internal/celutils/errors.go

```diff
@@ -20,3 +20,24 @@ func (e *ErrorCELParse) Is(target error) bool {
 func (e *ErrorCELParse) Unwrap() error {
 	return e.inner
 }
+
+// ErrorCELEval represents an error during CEL evaluation.
+type ErrorCELEval struct {
+	inner error
+}
+
+func (ErrorCELEval) Error() string {
+	return "error evaluating CEL expression"
```

Comment:
Would it help to have a static list of errors consolidated somewhere instead of a function returning a hard-coded (error) string?